### ISSUE TITLE
Include a link to this repository in the bot comment?

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [github/accessibility-alt-text-bot#40]
Right now, the bot's text is roughly:

> Uh oh! @{{ author }}, the image you shared is missing helpful alt text. Check {{ link }}.
>
> Alt text is an invisible description that helps screen readers describe images to blind or low-vision users. If you are using markdown to display images, add your alt text inside the brackets of the markdown image.
>
> Learn more about alt text at [Basic writing and formatting syntax: images on GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images).

...but nowhere does that link to this repository. I think it'd be useful for discoverability to immediately let people know what tooling is adding that message. I'd be most folks won't spend the time to dig into the repo's workflows to find it. Thoughts?
